pallets/gas/src/mock.rs

```diff
@@ -55,7 +56,7 @@ pallet_gear_gas::impl_config!(Test);
 common::impl_pallet_balances!(Test);
 
 parameter_types! {
-    pub const BlockHashCount: BlockNumber = 250;
+    pub const BlockHashCount: BlockNumber = BLOCK_HASH_COUNT;
```

Comment:
ah, whats what it is. revert the change pls, it doesnt relate to gear protocol and gear core